### ISSUE TITLE
[Tabs] remove outline from click

### ISSUE THREAD

**Issue by DanielRuf** [zurb/foundation-sites#10798]
The outline that is automatically added when a tab is clicked is very confusing.

So far we have to remove the outlines from the elements.

It would be good to have an option to remove the blue focus ring using a setting.

**Comment by johnuphoff** [comment]
For tabs you can add the outline: none; rule to the <a> within the tab (or globally) to remove the blue outline as you mentioned.  Foundation is designed to be accessible, so for keyboard users and others with reduced vision you may want to build fallback UI for tabs that indicate current focus (such as underline or color changes). It is not likely that zurb will add a flag to removes these outlines in the near future (or ever) but you could extend foundation to do this easily yourself.

If you want to learn more about accessibility - check out a11project:
https://a11yproject.com/posts/never-remove-css-outlines/

Also this talk by @ryanflorence at MountainWest Javascript - https://www.youtube.com/watch?v=YuzS-g6Qvq8

**Comment by DanielRuf** [comment]
Right. But the blue focus ring is ugly. I'm styling hover and focus states.

**Comment by DanielRuf** [comment]
Removing it is totally ok according to W3C and others when you apply some clear focus styles.

**Comment by DanielRuf** [comment]
The title does not reflect that it is ok. But says so in the 3 points. So the word 'never' is misleading imho.

**Comment by derick-montague** [comment]
There is a CSS 4 pseudo-class `:focus-ring` that is currently only being shipped in Firefox. This differentiates between mouse and keyboard focus on different elements. This can make the experience better for mouse users. Until it is ready to ship in all browsers, you can use the[ focus-ring polyfill](https://github.com/WICG/focus-ring). 

After including the polyfill, just add this ruleset to the style sheet. 
```
:focus:not('.focus-ring') {
  outline: none;
}
```

**Comment by DanielRuf** [comment]
Why not `:focus` and style it? JS polyfills are the very last solution.

So far the automatic focus ring after the click is irritating. There is already the different background color and other components also don't get it by clicking on elements.

**Comment by DanielRuf** [comment]
That's how it looks directly after the click.
![bildschirmfoto 2017-12-08 um 08 25 55](https://user-images.githubusercontent.com/827205/33755529-77a22b40-dbf1-11e7-9a1b-cb5fec8a65a8.png)

**Comment by derick-montague** [comment]
Completely agree regarding the polyfill. If you wanted to style it, my first approach would be to remove the outline and replace it on `:focus` with something like this. I think that the focus states for elements should be consistent, so if outlines are used for `<a>` and '<button>`, this solution allows you to maintain consistency, but keep the focus state contained within the tab.
```
box-shadow: inset 0 0 0 3px blue;
```

**Comment by DanielRuf** [comment]
I still think it is a bug, see https://github.com/zurb/foundation-sites/issues/7983

**Comment by ncoden** [comment]
No `outline` should be show when it is not for accessibility reasons, since it is not part of the Foundation design. So when `what-input` detect a mouse, we should have `outline: 0;` on the tab links. This is currently missing, I'll add it.

**Comment by DanielRuf** [comment]
awesome, thanks =)
